<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssemblyLoader</class>
 <widget class="QMainWindow" name="AssemblyLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Assembly Loader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="fileControlsLayout">
      <item>
       <widget class="QPushButton" name="loadFileButton">
        <property name="text">
         <string>Load Assembly File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>No file loaded</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextEdit" name="assemblyTextEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
      <property name="placeholderText">
       <string>Assembly code will appear here...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="steppingControlsLayout">
      <item>
       <widget class="QPushButton" name="stepButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sendInstructionButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Send Current Instruction</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="currentInstructionLabel">
      <property name="styleSheet">
       <string notr="true">font-weight: bold; padding: 5px; background-color: #f0f0f0; border: 1px solid #ccc;</string>
      </property>
      <property name="text">
       <string>Current Instruction: None</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>